core/server/api/shared/http.js

```diff
@@ -6,6 +6,22 @@ const http = (apiImpl) => {
     return (req, res, next) => {
         debug('request');
 
+        let apiKey = null;
+        let integration = null;
+        let user = null;
+
+        if (req.api_key) {
+            apiKey = {
+                id: req.api_key.get('id'),
+                type: req.api_key.get('type')
+            };
+            integration = req.api_key.get('integration_id');
```

Comment:
@kirrg001 integration will now be filled out with only an id for now just like the user is. Let me know if this is enough of info for you to continue with x_by changes? Otherwise merging these changes :)